### PR TITLE
Fix namespace, identifier-type and system per issue #773

### PR DESCRIPTION
**Pull request by Rene2mt**
This PR makes website documentation updates to resolve issue GSA/fedramp-automation/issues/773 

Specifically, it makes the following changes throughout the site:
- For FedRAMP extension properties, changes namespace from "**https**://fedramp.gov/ns/oscal" to "**http**://fedramp.gov/ns/oscal"
- For the SSP `identifier-type`, changes the value from "**https**://fedramp.gov/ns/oscal" to "**http**://fedramp.gov/ns/oscal"
- For POA&M facets, changes the `system` flag from "**https**://fedramp.gov" to "**http**://fedramp.gov/ns/oscal"

### CODE REVIEW

File: themes/uswds-hugo
Comment (aj-stein-gsa) [inline themes/uswds-hugo:1]: Blocking: can we back out submodules changes unless there is a pressing reason you can explain it in a follow-up comment?

**Review by aj-stein-gsa — CHANGES_REQUESTED** [review]
I would approve with the quote issue, but I wary of moving up submodules for site components with content edits unless there is a reason.

File: themes/uswds-hugo
Comment (Rene2mt) [reply themes/uswds-hugo:1]: will do...I unintentionally committed a submodule change

**Review by aj-stein-gsa — APPROVED** [review]

**Review by Gabeblis — APPROVED** [review]

**Comment by aj-stein-gsa** [comment]
If you are a community member subscribed to notifications outside of the developer team, be advised we are readying this for merge and publication. An email will be sent to pilot participants as well when this is complete, but this is a heads up that process is starting.

**Pull request merged**